test/e2e/aws.go

```diff
@@ -23,10 +27,16 @@ type awsTestHelper struct {
 	secretKey string
 }
 
-func newAWSHelper(awsAccessKeyID, awsSecretAccessKey string) (providerTestHelper, error) {
+func newAWSHelper(isOpenShiftCI bool, kubeClient client.Client) (providerTestHelper, error) {
+	awsAccessKeyID, awsSecretAccessKey, err := prepareConfigurations(isOpenShiftCI, kubeClient)
```

Comment:
`prepareConfigurations` can be a method of `awsTestHelper`, the same way you did for azure helper. Inside you can set  `keyID` and `secretKey` fields of the helper.
This way you'll be consistent in all the helpers and won't need the special name for each method like `preapreGCPConfigurations` for GCP.

Reply:
Yes andery, this is option one (bring under data member functions.)